db/migrate/20180914124755_create_books.rb

```diff
@@ -2,11 +2,11 @@ class CreateBooks < ActiveRecord::Migration[5.1]
   def change
     create_table :books do |t|
 
-      t.string :gender, null:false
```

Comment:
Beware of changing the migration directly, You should have created a new migration to change de DB column name, if you change the last migration directly then you have to de a rollback and run the old migration again which may cause troubles in the DB. Now the DB is short but in future PR please make a new migration to change the table

Reply:
Oh yes, i dont see that. In the future  i'll try not to make the same mistake